lte/gateway/python/integ_tests/s1aptests/test_attach_detach_secondary_pdn_with_dedicated_bearer_deactivate.py

```diff
@@ -29,9 +29,25 @@ def test_seconday_pdn_conn_ded_bearer_deactivate(self):
         dedicated bearers + detach"""
         num_ues = 1
         self._s1ap_wrapper.configUEDevice(num_ues)
+        req = self._s1ap_wrapper.ue_req
+
+        # APN details to be configured
+        ims = [
+            "ims",  # APN-name
+            5,  # qci
+            15,  # priority
+            0,  # preemption-capability
+            0,  # preemption-vulnerability
+            200000000,  # MBR UL
+            100000000,  # MBR DL
+        ]
+
+        apn_list = [ims]
+        self._s1ap_wrapper.configAPN(
+            "IMSI" + "".join([str(i) for i in req.imsi]), apn_list
+        )
```

Comment:
This function should only have the apn_list in its argument. The configuration of subscribers with the apns should happen in configUEDevice function above.

Reply:
This test case also leaves a dirty state behind as default bearers are not cleaned up.

Reply:
This TC is not updated as per the new APN config as it was failing on master when I created the PR. I will update now